raw-spans-grouper/helm/values.yaml

```diff
@@ -107,6 +107,7 @@ rawSpansGrouperConfig:
         numStreamThreads: 4
         bootstrapServers: "bootstrap:9092"
         schemaRegistryUrl: "http://schema-registry-service:8081"
+        cacheMaxBytesBuffering: 16777216
```

Comment:
```suggestion
        cacheMaxBytesBuffering: 16777216 # 16MB 
```

Reply:
I'd really like to see a comment here on why 16MB is better than 10MB? Why not 24MB or 32MB?  This is not purely ornamental information, it is to give the context to someone in the future on why 16MB was chosen (what problem was solved by increasing and why exactly that number) in case this has to be changed.

Reply:
16MB is not much different from 10MB. 10MB is the default value in Kafka streams. Just trying to standardize it to multiples of 16 for easier calculation. All other default mem configs are in multiples of 16 in hypertrace. The main idea here is to templatize only in helm charts so that this can be tuned based on the environment. At high scale depending on requirement this can be tuned  to high values like 1GB as well.

For more info
https://kafka.apache.org/10/documentation/streams/developer-guide/config-streams.html

Reply:
Thanks @laxmanchekka for the context. I don't prefer 10MB over 16MB just looking for context. I will add some of this notes in the yaml.